src/main/java/codechicken/enderstorage/container/ContainerEnderItemStorage.java

```diff
@@ -93,4 +90,23 @@ public void onContainerClosed(EntityPlayer entityplayer) {
         super.onContainerClosed(entityplayer);
         chestInv.closeInventory();
     }
+
+    @Override
+    public boolean canInteractWith(EntityPlayer entityplayer) {
+        return chestInv.isUsableByPlayer(entityplayer);
+    }
```

Comment:
I believe Intellij reformatted it to group it with the other overridden methods from `Container`, in the same order that they were declared in `Container`.